app.js

```diff
@@ -0,0 +1,44 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const bodyParser = require("body-parser");
+
+const usersRouter = require("./routes/users");
+const cardsRouter = require("./routes/cards");
+
+const { PORT = 3000 } = process.env;
+const app = express();
+
+mongoose.connect("mongodb://localhost:27017/mestodb", {
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  useFindAndModify: false,
+  useUnifiedTopology: true,
+});
+
+app.use((req, res, next) => {
+  req.user = {
+    _id: "5ecb8bf13a5aeb4efe09317e", //  _id созданного тестового пользователя
+  };
+  next();
+});
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+app.use("/users", usersRouter);
+app.use("/cards", cardsRouter);
+
+app.use("*", (req, res) => {
+  res.status(404).json({ message: "Запрашиваемый ресурс не найден" });
+});
+
+app.use((err, req, res) => {
+  console.error(err);
+  return res
+    .status(err.status)
```

Comment:
Нужно исправить: не работает обработчик ошибок.
Нужно обработать ситуацию для ошибок, у которых не указан код `status`:
```suggestion
    .status(err.status || 500)
```